SECP-service/src/main/java/com/visucius/secp/models/Message.java

```diff
@@ -0,0 +1,64 @@
+package com.visucius.secp.models;
+
+import javax.persistence.*;
+import java.util.Date;
+
+
+@Entity
+@Table(name = "message")
+public class Message {
+
+    @Id
+    @Column(name = "message_id", unique = true, nullable = false)
+    private int messageId;
+
+    @Column(name = "timestamp")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date dateTime;
+
+    @Column(name = "body")
+    private String messageBody;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+
```

Comment:
I think you are missing one key table, message_recipient. This table should have a message id and the user_id of receiver. It's almost similar to what i did with the user and roles.